utils.go

```diff
@@ -139,3 +142,26 @@ loop:
 	}
 	return results, err
 }
+
+func ShowMasterStatusBinlogPosition(db *sql.DB) (mysql.Position, error) {
+	row := db.QueryRow("SHOW MASTER STATUS")
+	var file string
+	var position uint32
+	var binlog_do_db, binlog_ignore_db, executed_gtid_set string
```

Comment:
all of these appear to be unused. can we get rid of them?

Reply:
No we cannot. `row.Scan` expects the same number of arguments as the columns returned. `SHOW MASTER STATUS` always return these columns and we don't really have an ability to filter them.